tests/accesstokens_test.go

```diff
@@ -40,6 +41,21 @@ func testRefreshTokenTest(t *testing.T) {
 	assert.Equal(t, token.ExpiresIn, newToken.ExpiresIn, "New access token's expiration is different from original one")
 }
 
+func testReferenceTokenTest(t *testing.T) {
+	// Create token
+	tokenParams := createRefreshableAccessTokenParams(testExpiredInSeconds)
+	token, err := testsAccessTokensService.CreateAccessToken(tokenParams)
+	assert.NoError(t, err)
+	// Refernece token
+	referenceTokenParams := createReferenceAccessTokenParams(token)
+	newToken, err := testsAccessTokensService.RefreshAccessToken(referenceTokenParams.CommonTokenParams)
+	assert.NoError(t, err)
+	// Validate
+	assert.NotEqual(t, token.AccessToken, newToken.AccessToken, "New access token is identical to original one")
+	assert.NotEqual(t, token.IsReferenceToken, newToken.IsReferenceToken, "New reference token is identical to original one")
```

Comment:
This fails the test. I'm not sure that the "include_reference_token" is included in the response from Xray.

https://github.com/jfrog/jfrog-client-go/actions/runs/4609825075/jobs/8147514091?pr=729#step:6:61